spring-xd-spark-streaming/src/main/java/org/springframework/xd/spark/streaming/examples/SparkLogModuleOptionsMetadata.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.xd.spark.streaming.examples;
+
+import org.springframework.xd.module.options.spi.ModuleOption;
+import org.springframework.xd.spark.streaming.DefaultSparkStreamingModuleOptionsMetadata;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+public class SparkLogModuleOptionsMetadata extends DefaultSparkStreamingModuleOptionsMetadata {
```

Comment:
I think it might be a bit of a chicken and egg problem, since we only know it's a Spark Streaming module based on the "spark" value for moduleExecutionFramework which is provided via the options. But if there's a cleaner alternative, we should do it... would involve changing the way we detect the Spark Streaming type of module.